cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -145,25 +145,33 @@ Hadoop/HBase Environment
 
 For a distributed enterprise, you must install these Hadoop components:
 
-+---------------+-------------------+---------------------------------------------+
-| Component     | Distribution      | Required Version                            |
-+===============+===================+=============================================+
-| **HDFS**      | Apache Hadoop DFS | 2.0.2-alpha or later                        |
-+               +-------------------+---------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.2.x or later or (HDP) 2.0 or later  |
-+---------------+-------------------+---------------------------------------------+
-| **YARN**      | Apache Hadoop DFS | 2.0.2-alpha or later                        |
-+               +-------------------+---------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.2.x or later or (HDP) 2.0 or later  |
-+---------------+-------------------+---------------------------------------------+
-| **HBase**     |                   | 0.94.2+, 0.96.0+, 0.98.0+                   |
-+---------------+-------------------+---------------------------------------------+
-| **Zookeeper** |                   | Version 3.4.3 or later                      |
-+---------------+-------------------+---------------------------------------------+
-| **Hive**      |                   | Version 12.0 or later                       |
-+               +-------------------+---------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.3.x or later or (HDP) 2.0 or later  |
-+---------------+-------------------+---------------------------------------------+
++---------------+-------------------+--------------------------------------------------+
+| Component     | Distribution      | Required Version                                 |
++===============+===================+==================================================+
+| **HDFS**      | Apache Hadoop DFS | 2.0.2-alpha through 2.5.0                        |
++               +-------------------+--------------------------------------------------+
+|               | CDH or HDP        | (CDH) 4.2.x through 5.3.3 or (HDP) 2.0 or 2.1    |
++---------------+-------------------+--------------------------------------------------+
+| **YARN**      | Apache Hadoop DFS | 2.0.2-alpha through 2.5.0                        |
++               +-------------------+--------------------------------------------------+
+|               | CDH or HDP        | (CDH) 4.2.x through 5.3.3 or (HDP) 2.0 or 2.1    |
++---------------+-------------------+--------------------------------------------------+
+| **HBase**     | Apache Hadoop DFS | 0.94.2.x, 0.96.0.x, and 0.98.6                   |
```

Comment:
hbase versions still not technically correct, i'd suggest: ``0.94.2+, 0.96.x, 0.98.x``